doc/zephyr.doxyfile.in

```diff
@@ -2370,6 +2370,7 @@ PREDEFINED             = __DOXYGEN__ \
                          CONFIG_NET_L2_ETHERNET_MGMT \
                          CONFIG_NET_MGMT_EVENT \
                          CONFIG_NET_SOCKETS_POSIX_NAMES \
+                         CONFIG_NET_CONNECTION_MANAGER \
```

Comment:
please use `__DOXYGEN__` instead (see some other headers)

Reply:
If I use `__DOXYGEN__`, then hyperlinks to some of the KConfigs do not work (because those KConfigs are not enabled during docs builds)

I'll try one more time to make sure this was not a fluke, but if not, how do you recommend I fix this issue if not by appending to the doxyfile?

Reply:
I was able to get it to work by including a __DOXYGEN__clause in the CMakeLists.txt file

For whatever reason, my local docs builds show errant behavior, so I will have to force-push a build without that addition just to confirm it is truly necessary.

Reply:
IDK. I removed it from the CMakeLists.txt, and the KConfig references still work. I'm not sure anymore what problem I was actually experiencing.